packages/clay-navigation-bar/stories/index.tsx

```diff
@@ -0,0 +1,134 @@
+/**
+ * © 2019 Liferay, Inc. <https://liferay.com>
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import ClayNavigationBar from '../src/index';
+import React from 'react';
+import {storiesOf} from '@storybook/react';
+
+import 'clay-css/lib/css/atlas.css';
+
+const spritemap = require('clay-css/lib/images/icons/icons.svg');
+
+storiesOf('ClayNavigationBar', module)
+	.add('default', () => (
```

Comment:
As we had discussed, you can already add the knobs in this PR to narrow down the list of examples. If the knob is not enough to show a particular use case, add a story.